tests/draft6/id.json

```diff
@@ -48,6 +48,67 @@
                 "valid": false
             }
         ]
+    },
+    {
+        "description": "non-schema object containing a plain-name $id property",
```

Comment:
They use `if`/`then`/`else` which weren't introduced to draft 7.  We need to remove these from draft 6.

Reply:
Would you mind opening an issue, I'll remove them if someone doesn't get to it first

Reply:
Fix in #562